patches/roslyn/0002-Add-solution-for-source-build.patch

```diff
@@ -0,0 +1,147 @@
+From c7dbba01272248423989a1816625dccb3e129b74 Mon Sep 17 00:00:00 2001
```

Comment:
This patch is unnecessary, this commit of roslyn has Sourcebuild.sln.

Reply:
Updated the sha to the commit that fixes SourceBuild.sln.